changelogs/fragments/openssh_cert-pkcs11.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - openssh_cert - add support for PKCS#11 tokens
```

```suggestion
  - "openssh_cert - add support for PKCS#11 tokens (https://github.com/ansible-collections/community.crypto/pull/95)."
```
The quotes are probably necessary since `#` starts comments in YAML files.